io/src/test/scala/sbt/internal/io/SourceModificationWatchSpec.scala

```diff
@@ -21,10 +21,9 @@ abstract class SourceModificationWatchSpec(
 
     IO.write(file, "foo")
 
-    // watchTest(parentDir) {
-    //   IO.write(file, "bar")
-    // }
-    pending // until fixed https://github.com/sbt/io/issues/82
+    watchTest(parentDir) {
+      IO.write(file, "bar")
+    }
```

Comment:
This shouldn't've been enabled like this because it fails on HFS+. See https://github.com/sbt/io/pull/102/files#diff-32eb449ec95851dd791288244207a8f8R32.